### DISCUSSION TITLE
RabbitMQ Server Pod Crash Loop BackOff on Azure AKS Cluster

### DISCUSSION THREAD

**Discussion by AlbertoBettaACN** [rabbitmq/rabbitmq-server#4052] [Questions]
When trying to install Rabbit MQ Server on Azure Kubernetes Cluster with Cluster Operator the Pod of the RabbitMQ Server gets stuck on Crash BackOff. 

This is the yaml used:

apiVersion: rabbitmq.com/v1beta1
kind: RabbitmqCluster
metadata:
  name: hello-world
  namespace: rabbitmq-system

It creates the statuful set but the Pod triggered by the Statefull Set gives this error ( by running kubectl logs hello-world-server-0 -n rabbitmq-system)

`WARNING: 'docker-entrypoint.sh' generated/modified the RabbitMQ configuration file, which will no longer happen in 3.9 and later! (https://github.com/docker-library/rabbitmq/pull/424)

Generated end result, for reference:

loopback_users.guest = false
total_memory_available_override_value = 2147483648
listeners.tcp.default = 5672
management.tcp.port = 15672

Configuring logger redirection
16:19:10.740 [warning] cluster_formation.randomized_startup_delay_range.min and cluster_formation.randomized_startup_delay_range.max are deprecated
16:19:10.947 [warning] Failed to write PID file "/var/lib/rabbitmq/mnesia/rabbit@hello-world-server-0.hello-world-nodes.rabbitmq-system.pid": permission denied
16:19:11.055 [error] Failed to create Ra data directory at '/var/lib/rabbitmq/mnesia/rabbit@hello-world-server-0.hello-world-nodes.rabbitmq-system/quorum/rabbit@hello-world-server-0.hello-world-nodes.rabbitmq-system', file system operation error: enoent
16:19:11.055 [error] Supervisor ra_sup had child ra_system_sup started with ra_system_sup:start_link() at undefined exit with reason {error,"Ra could not create its data directory. See the log for details."} in context start_error
16:19:11.056 [error] CRASH REPORT Process <0.248.0> with 0 neighbours exited with reason: {error,"Ra could not create its data directory. See the log for details."} in ra_system_sup:init/1 line 43
16:19:11.056 [error] CRASH REPORT Process <0.242.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,ra_system_sup,{error,"Ra could not create its data directory. See the log for details."}}},{ra_app,start,[normal,[]]}} in application_master:init/4 line 142
{"Kernel pid terminated",application_controller,"{application_start_failure,ra,{{shutdown,{failed_to_start_child,ra_system_sup,{error,\"Ra could not create its data directory. See the log for details.\"}}},{ra_app,start,[normal,[]]}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,ra,{{shutdown,{failed_to_start_child,ra_system_sup,{error,"Ra could not create its data directory. See the log for details."}

Crash dump is being written to: erl_crash.dump...
`

**Comment by nathanjbrockway** [comment]
Don't use the rabbitmq-system as your namespace. Instead use default or another namespace you prefer.
